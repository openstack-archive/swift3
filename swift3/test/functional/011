#!/bin/bash

# Upload Part

. ./common

num=10

S3USER=tester

_s3_put /bucket

upload_id=$(_s3_post /bucket/object?uploads \
    -H "x-amz-acl: public-read-write" \
    -H "x-amz-meta-foo1: 1" \
    -H "x-amz-meta-foo2: 2" \
    -H "x-amz-meta-foo3: 3" | \
    _xq /InitiateMultipartUploadResult/UploadId)

for i in `seq $num`; do
    truncate -s 512K $TEST_DIR/segment_50M.dat_$i
    _s3_put /bucket/object?uploadId=${upload_id}\&partNumber=$i \
        -T $TEST_DIR/segment_50M.dat_$i
done

for i in `seq $num`; do
    j=$(($i - 1))
    rm $TEST_DIR/segment_50M.dat_$i
done

