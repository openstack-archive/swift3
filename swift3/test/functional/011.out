QA output created by 011
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X POST -H x-amz-acl: public-read-write -H x-amz-meta-hoge1: 1 -H x-amz-meta-hoge2: 2 -H x-amz-meta-hoge3: 3 http://SWIFT_HOST/bucket/object?uploads... 200
> s3curl --id tester -- -X PUT -T ./segment_50M.dat_00 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=1... 200
> s3curl --id tester -- -X PUT -T ./segment_50M.dat_01 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=2... 200
> s3curl --id tester -- -X PUT -T ./segment_50M.dat_02 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=3... 200
> s3curl --id tester -- -X PUT -T ./segment_50M.dat_03 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=4... 200
> s3curl --id tester -- -X PUT -T ./segment_50M.dat_04 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=5... 200
> s3curl --id tester -- -X PUT -T ./segment_50M.dat_05 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=6... 200
> s3curl --id tester -- -X PUT -T ./segment_50M.dat_06 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=7... 200
> s3curl --id tester -- -X PUT -T ./segment_50M.dat_07 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=8... 200
> s3curl --id tester -- -X PUT -T ./segment_50M.dat_08 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=9... 200
> s3curl --id tester -- -X PUT -T ./segment_50M.dat_09 http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=10... 200
